mods/gen2/moves.js

```diff
@@ -12,13 +12,17 @@ exports.BattleMovedex = {
 			if (target.boosts.atk >= 6) {
 				return false;
 			}
+
+			var boost = 0;
+
 			if (target.hp <= target.maxhp / 2) {
-				this.boost({atk: 2});
-				return false;
+				boost = (target.boost ? target.boost.atk : 0) + 2;
```

Comment:
Technically different, but yes. That will work probably, although it will send `-boost|POKEMON|atk|2|[from] ` and I don't know how the client will handle that.